cmd/head/head.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/codescalersinternships/Coreutils-Fatma-Ebrahim/internal/head"
+)
+
+func main() {
+	n := 10
```

Comment:
value here is not relevant since it is set according to the default you set in IntVar or the user input. You can just leave it as `var n int`